tools/appliance/systemvmtemplate/scripts/configure_proxy.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
This is not needed for the new system, look at building with packer and how to export http proxy.

Reply:
@fmaximus ^^